# PointAndMechaState.msg

float64 x  # X座標
float64 y  # Y座標
float64 angle  # ロボットの向き（ラジアンまたは度）
MechaState command  # そのポイントでのコマンド